atomics/T1070.003/T1070.003.yaml

```diff
@@ -94,3 +94,24 @@ atomic_tests:
        hostname
        whoami
     name: sh
+- name:  Disable Bash History Logging with SSH -T
+  description: |
+    Keeps history clear and stays out of lastlog,wtmp,btmp ssh -T keeps the ssh client from catching a proper TTY, which is what usually gets logged on lastlog
+  supported_platforms:
+  - linux
```

Comment:
This technique will certainly work on `macos` and other flavors of Linux, however as-is it will only run on `RPM` based distributions.